draft-ietf-quic-transport.md

```diff
@@ -865,92 +865,87 @@ endpoint, as described in {{termination}}.
 ## Matching Packets to Connections {#packet-handling}
 
 Incoming packets are classified on receipt.  Packets can either be associated
-with an existing connection, be discarded, or - for servers - potentially create
-a new connection.
-
-Packets that can be associated with an existing connection are handled according
-to the current state of that connection.  Packets are associated with existing
-connections using connection ID if it is present; this might include connection
-IDs that were advertised using NEW_CONNECTION_ID ({{frame-new-connection-id}}).
-Packets without connection IDs and long-form packets for connections that have
-incomplete cryptographic handshakes are associated with an existing connection
-using the tuple of source and destination IP addresses and ports.
-
-A packet that uses the short header could be associated with an existing
-connection with an incomplete cryptographic handshake.  Such a packet could be a
-valid packet that has been reordered with respect to the long-form packets that
-will complete the cryptographic handshake.  This might happen after the final
-set of cryptographic handshake messages from either peer.  These packets are
-expected to be correlated with a connection using the tuple of IP addresses and
-ports.  Packets that might be reordered in this fashion SHOULD be buffered in
-anticipation of the handshake completing.
-
-0-RTT packets might be received prior to a Client Initial packet at a server.
-If the version of these packets is acceptable to the server, it MAY buffer these
-packets in anticipation of receiving a reordered Client Initial packet.
+with an existing connection, or - for servers - potentially create a new
+connection.
+
+Hosts handle packets that can be associated with an existing connection
+according to the current state of that connection. Both short form packets
+without connection IDs and long-form packets for connections that have
+incomplete cryptographic handshakes are associated with an existing
+connection using the tuple of source and destination IP addresses and ports.
+Other packets are associated with existing connections using connection ID
+the connection ID in the header; this might include connection IDs that were
+advertised using NEW_CONNECTION_ID ({{frame-new-connection-id}}).
+
+Clients SHOULD discard any packet that cannot be associated with an existing
```

Comment:
This paragraph is very odd right here.  This seems to imply connections are never created.  Possibly remove this paragraph?

Reply:
The key word here is "Clients". This finishes up instructions for clients, and the rest of the section is about servers. That said, a week after I wrote this it took me a minute to notice that. Any suggestions on how to make the word "clients" and its precise meaning more salient?